app/js/components/manageApps/ManageApps.jsx

```diff
@@ -76,15 +76,21 @@ export default class ManageApps extends React.Component {
     });
   }
 
-  handleUpload() {
+  handleUpload(file = null) {
     const applicationDistribution = location.href.split('/')[2];
     const url = location.href.split('/')[3];
     let apiBaseUrl = `/${applicationDistribution}/${url}/ws/rest`;
     this.requestUrl ='/owa/addapp';
     let that = this;
 
+    
     const addonFile = new FormData();
-    addonFile.append('file', document.getElementById('fileInput').files[0]);
+
+    if(file!= null) {
```

Comment:
Hi, I would recommend using `!==` as opposed to `!=` here.

Reply:
Okay. Changing this.